src/components/ElevatedButton/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface ElevatedButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  isFullSize: boolean;
+}
+
+const ElevatedButton = ({ children, isFullSize, ...attributes }: ElevatedButtonProps): JSX.Element => {
+  const fullWidth = isFullSize ? 'w-full' : '';
+  return (
+    <button
+      type="button"
+      className={`bg-white text-black-chinese font-bold text-regular tracking-survey-tight rounded-[10px] focus:outline-none focus:shadow-outline h-14 ${fullWidth}`}
```

Comment:
let's use [classnames](https://github.com/JedWatson/classnames) library here, it'll simplify using classnames a lot 😉 
For example, this code can be re-written as:
```suggestion
            className={classNames(
              'bg-white text-black-chinese font-bold text-regular tracking-survey-tight rounded-[10px] focus:outline-none focus:shadow-outline h-14',
              { 'w-full': isFullSize },
            )}
```

Reply:
Also, we could extract the class names to a variable for better readability:

```js
const DEFAULT_CLASS_NAMES = 'bg-white text-black-chinese font-bold text-regular tracking-survey-tight rounded-[10px] focus:outline-none focus:shadow-outline h-14'
```

Reply:
TIL, thanks, @tyrro